frontend/src/components/jobs/search/search-bar.tsx

```diff
@@ -5,8 +5,22 @@ import { IconSearch } from "@tabler/icons-react";
 export default function SearchBar() {
   return (
     <Input
-      placeholder="Search for a company or a role..."
-      leftSection={<IconSearch size={16} />}
+      placeholder="Search for a company or a role..." // not my proudest workaround
```

Comment:
what do you mean by workaround 😭

Reply:
my bad, I had a funky workaround but then I did it properly forgot to remove the comment :grin:  
```suggestion
      placeholder="Search for a company or a role..."
```